website/content/api-docs/system/rotate/index.mdx

```diff
@@ -0,0 +1,54 @@
+---
+sidebar_label: Overview
+description: >- 
+  The '/sys/rotate' endpoints are used to rotate unseal, root, backend encryption keys.
+---
+
+# `/sys/rotate`
+
+### Encryption key rotation
+
+The `/sys/rotate/keyring` endpoint is used to rotate the encryption key. It is
+also possible to configure automatic encryption key rotation using
+`/sys/rotate/keyring/config` endpoint.
+
+### Rotate unseal/recovery keys
+
+The `/sys/rotate/root` endpoints are used to rotate the unseal keys for OpenBao.
```

Comment:
```suggestion
The `/sys/rotate/root` endpoints are used to rotate the root key for OpenBao
and the unseal key shares when using Shamir's manual unseal mechanism.
```

Reply:
Note that there's always the root key even with Shamir's; Shamir's combined secret encrypts the root key, just like with an auto-unseal mechanism.